packages/geo/src/patterning.ts

```diff
@@ -0,0 +1,41 @@
+// __private-exports
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const nomatch = (t: string) => new RegExp(`___UNMATCHED_${t}_TOKEN___`);
+
+export const Patterning = {
```

Comment:
This object wrapping isn't really necessary in a module based system. The module itself is your object and your encapsulation. And forcing the grouping can have downstream implications.

More of an "open a discussion" addition, I'm very much one for *not* adding dep's willy-nilly but I believe this is just for constructing regex's (from a brief skim). Would it be better long term to just use something like [ts-regex-builder](https://www.npmjs.com/package/ts-regex-builder), [super-expressive](https://github.com/francisrstokes/super-expressive) or similar?

Reply:
I converted these over to exporting individual function instead. I'll finish some other changes based on feedback before pushing changes.